dll/win32/shell32/shlexec.cpp

```diff
@@ -2031,6 +2084,18 @@ static BOOL SHELL_execute(LPSHELLEXECUTEINFOW sei, SHELL_ExecuteW32 execfunc)
         return retval > 32;
     }
 
+    if (!(sei_tmp.fMask & SEE_MASK_IDLIST) && // Not an ID List
+        lstrcmpiW(PathFindExtensionW(sei_tmp.lpFile), L".LNK") != 0) // Not a shortcut file
+    {
+        CComHeapPtr<ITEMIDLIST> pidlParsed;
+        HRESULT hr = SHParseDisplayName(sei_tmp.lpFile, NULL, &pidlParsed, SFGAO_STORAGECAPMASK, NULL);
```

Comment:
Why are you passing SFGAO_STORAGECAPMASK if you don't care about the returned attributes?

Reply:
Used zero.